### ISSUE TITLE
Keras - Activations

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#705]
#  Activations
激活函数可以通过设置单独的激活层实现，也可以在构造层对象时通过传递activation参数实现

## 预定义激活函数
- [ ] softmax
- [ ] elu
- [ ] selu
- [ ] softplus
- [ ] softsign
- [ ] relu
- [ ] tanh
- [ ] sigmoid
- [ ] hard_sigmoid
- [ ] linear
- [ ] [高级激活函数](https://keras.io/zh/layers/advanced-activations/)

**Comment by junxnone** [comment]
#9
